I see why I did the refactoring. It was because mypy would fail without it 🤔

```
homeassistant/components/peco/__init__.py:66: error: Incompatible return value type (got "PECOCoordinatorData", expected "OutageResults")  [return-value]
homeassistant/components/peco/__init__.py:99: error: Argument "update_method" to "DataUpdateCoordinator" has incompatible type "Callable[[], Coroutine[Any, Any, bool]]"; expected "Callable[[], Awaitable[OutageResults]] | None"  [arg-type]
Found 2 errors in 1 file (checked 5 source files)
```

--- a/homeassistant/components/peco/__init__.py
+++ b/homeassistant/components/peco/__init__.py
@@ -49,7 +49,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     # Outage Counter Setup
     county: str = entry.data[CONF_COUNTY]
 
-    async def async_update_outage_data() -> OutageResults:
+    async def async_update_outage_data() -> PECOCoordinatorData:
         """Fetch data from API."""
         try:
             outages: OutageResults = (
@@ -65,7 +65,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             raise UpdateFailed(f"Error parsing data: {err}") from err
         return data
 
-    coordinator = DataUpdateCoordinator(
+    outage_coordinator = DataUpdateCoordinator(
         hass,
         LOGGER,
         name="PECO Outage Count",
@@ -73,9 +73,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         update_interval=timedelta(minutes=OUTAGE_SCAN_INTERVAL),
     )
 
-    await coordinator.async_config_entry_first_refresh()
+    await outage_coordinator.async_config_entry_first_refresh()
 
-    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {"outage_count": coordinator}
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {
+        "outage_count": outage_coordinator
+    }
 
     if phone_number := entry.data.get(CONF_PHONE_NUMBER):
         # Smart Meter Setup]
@@ -92,7 +94,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
                 raise UpdateFailed(f"Error parsing data: {err}") from err
             return data
 
-        coordinator = DataUpdateCoordinator(
+        meter_coordinator = DataUpdateCoordinator(
             hass,
             LOGGER,
             name="PECO Smart Meter",
@@ -100,9 +102,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             update_interval=timedelta(minutes=SMART_METER_SCAN_INTERVAL),
         )
 
-        await coordinator.async_config_entry_first_refresh()
+        await meter_coordinator.async_config_entry_first_refresh()
 
-        hass.data[DOMAIN][entry.entry_id]["smart_meter"] = coordinator
+        hass.data[DOMAIN][entry.entry_id]["smart_meter"] = meter_coordinator
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
     return True

--- a/homeassistant/components/peco/manifest.json
+++ b/homeassistant/components/peco/manifest.json
@@ -5,5 +5,5 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/peco",
   "iot_class": "cloud_polling",
-  "requirements": ["peco==0.0.29"]
+  "requirements": ["peco==0.0.30"]
 }

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -1543,7 +1543,7 @@ panasonic-viera==0.3.6
 pdunehd==1.3.2
 
 # homeassistant.components.peco
-peco==0.0.29
+peco==0.0.30
 
 # homeassistant.components.pencom
 pencompy==0.0.3

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -1240,7 +1240,7 @@ panasonic-viera==0.3.6
 pdunehd==1.3.2
 
 # homeassistant.components.peco
-peco==0.0.29
+peco==0.0.30
 
 # homeassistant.components.escea
 pescea==1.0.12

